lib/utils/runBonjour.js

```diff
@@ -2,9 +2,10 @@
 
 function runBonjour({ port }) {
   const bonjour = require('bonjour')();
+  const os = require('os');
 
   bonjour.publish({
-    name: 'Webpack Dev Server',
+    name: `Webpack Dev Server ${os.hostname()}:${port}`,
```

Comment:
Just questions - this names is used somewhere?

Sorry, i am not familiar with `bonjour`,

Reply:
bonjour is used to broadcast services in current network, and it's used as service name

Reply:
maybe better use `hostname` from here https://github.com/webpack/webpack-dev-server/blob/61b949e076b887734d318889e8a6420eccd99404/lib/Server.js#L743

Reply:
I've check it, and it's set to 'localhost' (and it will be in 9/10 cases) so it could still colide with some other webpack-dev-server in local network.

Reply:
@roblan port should be difference in this case, you can't run multiple webpack-dev-server using the same port

Reply:
@evilebottnawi not on the same machine, but this name should be unique across local network.

Reply:
:+1: